src/pages/profile.js

```diff
@@ -1,12 +1,34 @@
 import React from 'react';
-import Footer from '../components/footer';
+import { Link } from 'react-router-dom';
 import Header from '../components/header';
+import Footer from '../components/footer';
+
+export default function Perfil() {
+  const user = JSON.parse(localStorage.getItem('user'));
+  const clearLocalStorage = () => {
+    localStorage.clear()
+  }
 
-export default function Profile() {
   return (
     <>
       <Header page="Perfil" />
-      <div data-testid="page-title">Perfil</div>
+      <h1 data-testid="profile-email">{ user?.email }</h1>
```

Comment:
```suggestion
      <h1 data-testid="profile-email">{ user.email }</h1>
```
Apenas para facilitar a localização desse erro... :blush: